templates/x86/qemu/mods.config

```diff
@@ -44,7 +44,7 @@ configuration conf {
 	include embox.kernel.thread.signal.siginfoq
 
 	@Runlevel(2) include embox.kernel.task.multi
-	@Runlevel(2) include embox.kernel.thread.core
+	@Runlevel(2) include embox.kernel.thread.core(thread_stack_size=16384)
```

Comment:
For me, it's OK to have 16K user stack on x86.
Stack protection should pay off this overhead.